tests/phpunit/includes/links-trait.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * Trait to ease tests made in directory as well as removing path from flag URL.
```

Comment:
Attempting to claify the comment
I do not understand the part related to the flag url. 
```suggestion
 * Trait to ease tests made with WP installed in a subfolder.
```